tests/test_merkle.py

```
@@ -0,0 +1,117 @@
+import unittest
+from core.merkle import MerkleTree, calculate_merkle_root
+
+
+class TestMerkleTree(unittest.TestCase):
+    def test_empty_transactions(self):
+        root = calculate_merkle_root([])
+        self.assertIsNone(root)
```

Ruff (0.15.1)

[warning] Use a regular `assert` instead of unittest-style `assertIsNone`

Replace `assertIsNone(...)` with `assert ...`

(PT009)